### DISCUSSION TITLE
Prisma create returns wrong types when a schema has multi level relations

### DISCUSSION THREAD

**Discussion by imevanc** [prisma/prisma#18191] [Q&A]
### Given the following schema

```prisma
generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider          = "postgresql"
  url               = env("DATABASE_URL")
}

model User {
  id             String      @id @default(cuid())
  name           String?
  email          String?     @unique
  tasks          Task[]
}

model Task {
  id        String   @id @default(cuid())
  createdAt DateTime @default(now())
  updatedAt DateTime @updatedAt
  title     String?
  User      User?    @relation(fields: [userId], references: [id])
  userId    String?
}
```

#### The created User type doesn't have a task property, so I had to create a UserWithTasks type which is extended by the User and the Task types. Shouldn't the User type created by Prisma include a task property? Cheers!

**Comment by ludralph** [comment]
Hi @imevanc 👋 

Did you run the command `npx prisma generate` to generate the types?

I ran your shared schema file and I got this
<img width="769" alt="image" src="https://user-images.githubusercontent.com/13855608/222664377-a66f328b-f204-46d9-a377-b1d3fde40d28.png">

The generated types had the task property.

**Comment by imevanc** [comment]
Hey @ludralph, 

I've just run it again and can confirm that the generated User type hasn't any Task property.
Any help?

Cheers,
Evan

**Reply by ludralph** [reply]
Can you share your repository so I can figure it what's going on?

**Reply by imevanc** [reply]
Cheers mate, please see that below. @ludralph 

https://github.com/imevanc/pet-pal

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
